.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/debian/.devcontainer/base.Dockerfile
```

Comment:
Should we add license header here as well?

Reply:
It is ok now